src/config/development.json

```diff
@@ -2,6 +2,7 @@
 	"css_url": "http://localhost:9000/happychat.css",
 	"happychat_url": "https://happychat-io-staging.go-vip.co/customer",
 	"twemoji_cdn_url": "https://s0.wp.com/wp-content/mu-plugins/wpcom-smileys/twemoji/2/",
+	"oauth_client_id": "54006",
```

Comment:
As per the comments https://github.com/Automattic/happychat-client/pull/188#discussion_r178524068 I guess we are also removing this?

Reply:
Indeed, it will be updated when merging the other branch.